### ISSUE TITLE
tablet file for the Turcom TS-6610

### ISSUE THREAD

**Issue by different55** [linuxwacom/libwacom#18]
The TS-6610 is just a re-branded Huion H610. Basically identical in every way (even the same IDs), but different enough that the Huion's .tablet file doesn't pick it up. I modified a copy of the Huion's file to change or erase the device/vendor names and that seems to do the trick.

libwacom picks it up perfectly and the settings mostly work. The "Left-Handed Orientation" checkbox doesn't work so I set that to false. The "Tracking Mode" dropdown also doesn't work but I don't know how to make that disappear.

```# Turcom
# TS-6610
#

[Device]
Name=Turcom TS-6610
DeviceMatch=usb:256c:006e
Class=Bamboo
Width=10
Height=6
IntegratedIn=
Layout=huion-h610-pro.svg
Styli=0xffffd;

[Features]
Stylus=true
Reversible=true
Touch=false
Buttons=8

[Buttons]
Left=A;B;C;D;E;F;G;H
# FIXME: This needs to be verified by someone with an actual device
EvdevCodes=0x100;0x101;0x102;0x103;0x104;0x105;0x106;0x107```

**Comment by whot** [comment]
You should be able to add `usb:256c:006e:Turcom TS-6610` to the `DeviceMatch` line of the existing `data/huion-h610-pro.tablet` file. See commit 937c6e61d5, the Huion tablets are notorious for that.

> The "Left-Handed Orientation" checkbox doesn't work so I set that to false.

The actual implementation of left-handed is left to other pieces in the stack, libwacom merely describes the device. So in this case the device is capable of being used in left-handed mode (i.e. it's not a symmetrical device). Whether something implements or the implementation works this doesn't matter for the libwacom tablet file.

> FIXME: This needs to be verified by someone with an actual device

Presumably you can verify this? :)

Please submit a PR with the DeviceMatch line extended for the 610 to match on your device too. And if you can verify the evdev codes too that'd be great. Run `evemu-record` against the device and compare the codes as you press the buttons in order. Thanks.

**Comment by different55** [comment]
Turns out the real name of this tablet is "Turcom TS-6610 *" While running evemu-record I saw the names "Turcom TS-6610 Pen", "Turcom TS-6610 Mouse", "Turcom TS-6610 Keyboard", "Turcom TS-6610 Consumer Control", and "Turcom TS-6610 Pad" on all the different event files. Near as I can tell only the Pen and Pad devices do much of anything.

Using the Pen and Pad names in the DeviceMatch seemed to make it work right. Even left handed orientation and the absolute/relative dropdown work just fine now.

Pressing the buttons top to bottom I see (in the 3rd column): 100, 101, 102, 103, 104, 105, 106, 107 so that all matches up there. I'll send a pull request the moment I figure out how. Thanks for all the help!
